clients/fides-js/src/components/LanguageSelector.tsx

```diff
@@ -0,0 +1,70 @@
+import { h } from "preact";
+import { I18n } from "../lib/i18n";
+import { useI18n } from "../lib/i18n/i18n-context";
+import { debugLog, FidesInitOptions } from "../fides";
+import MenuItem from "./MenuItem";
+import { FIDES_A11Y_CONSTANTS } from "../lib/consent-constants";
+
+interface LanguageSelectorProps {
+  i18n: I18n;
+  options: FidesInitOptions;
+}
+
+const LanguageSelector = ({ i18n, options }: LanguageSelectorProps) => {
+  const { currentLocale, setCurrentLocale } = useI18n();
+
+  const handleLocaleSelect = (locale: string) => {
+    if (locale !== i18n.locale) {
+      i18n.activate(locale);
+      setCurrentLocale(locale);
+      document.getElementById(FIDES_A11Y_CONSTANTS.FIDES_REFOCUS_EL)?.focus();
```

Comment:
Can we name the el something more generic instead of being tied to the functionality we add in the js? Main reason is when we add DOM ids and classes and such, I want to ensure it's super future-proof